build.gradle

```diff
@@ -40,6 +40,7 @@ googleJavaFormat {
   exclude 'build/**'
   exclude 'bazel*/**'
 }
+verifyGoogleJavaFormat.onlyIf { JavaVersion.current().isJava8Compatible() }
```

Comment:
Java 11 also works fine for the plugin. Therefore, leveraging JavaVersion's isJava8Compatible method.

```
suztomo-macbookpro44:gax-java suztomo$ java -version
java version "11.0.4" 2019-07-16 LTS
Java(TM) SE Runtime Environment 18.9 (build 11.0.4+10-LTS)
Java HotSpot(TM) 64-Bit Server VM 18.9 (build 11.0.4+10-LTS, mixed mode)

suztomo-macbookpro44:gax-java suztomo$ ./gradlew verifyGoogleJavaFormat

Deprecated Gradle features were used in this build, making it incompatible with Gradle 5.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/4.9/userguide/command_line_interface.html#sec:command_line_warnings

BUILD SUCCESSFUL in 6s
5 actionable tasks: 5 executed
```